File: verify/verify/ruby/check.rb
require "net/http"
require "uri"

uri = URI.parse("https://api.nexmo.com/verify/check/json")
params = {
    "api_key" => API_KEY,
    "api_secret" => API_SECRET,
    "request_id" => "31d724da3286461196a82ef3eec62d2a",
    "code" => "4345"
}

response = Net::HTTP.post_form(uri, params)

puts response.body

File: verify/verify/python/check.py
import urllib

params = {
    'api_key': API_KEY,
    'api_secret': API_SECRET,
    'request_id': 'd945c3655d9043c0b0b3941e1144f258',
    'code': '1665'
}

url = 'https://api.nexmo.com/verify/check/json?' + urllib.urlencode(params)

response = urllib.urlopen(url)
print response.read()
